docs/howtos/development.md

```diff
@@ -43,30 +43,33 @@ specific instructions, there are some differences for dev environments.
 These differences are explained in the section [Behind the curtains](#curtain) at the end of this document.
 
 Since we use `k3d` in our CI tests we have created the make target `prepare_environment_k3d` to prepare
-such an environment. For all other environments the following commands should be sufficient:
+such an environment. That script will use the value of the "EPINIO_SYSTEM_DOMAIN" environment variable
+for the `system-domain` installation argument. If that is not set, it will try to use a "magic" domain
+in the form of "1.2.3.4.omg.howdoi.website" where `1.2.3.4` is the IP address of your k3d cluster and
+`omg.howdoi.website` is a mirror-dns service which resolves to the IP address in front of it (similar to nip.io, xip.io etc).
+
+For all other environments the following commands should be sufficient:
 
 ```
-EPINIO_DONT_WAIT_FOR_DEPLOYMENT=1 ./dist/epinio-linux-amd64 install --skip-default-namespace
+EPINIO_DONT_WAIT_FOR_DEPLOYMENT=1 ./dist/epinio-linux-amd64 install --system-domain=your-system-domain-here.org
 make patch-epinio-deployment
-./dist/epinio-linux-amd64 namespace create workspace
-./dist/epinio-linux-amd64 target workspace
 ```
 
+You should use a `system-domain` that points to the Traefik Service on your cluster.
```

Comment:
`should`? Shouldn't this be `have to` ? Because if you don't the installation breaks ?
```suggestion
You have to use a `system-domain` that points to the Traefik Service on your cluster.
```